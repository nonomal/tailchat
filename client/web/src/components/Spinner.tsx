import { Icon } from 'tailchat-design';
import React from 'react';

export const Spinner: React.FC = React.memo(() => {
  return <Icon className="animate-spin mr-3 inline" icon="mdi:loading" />;
});
Spinner.displayName = 'Spinner';
