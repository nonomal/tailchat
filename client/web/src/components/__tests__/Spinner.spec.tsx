import { render } from '@testing-library/react';
import React from 'react';
import { Spinner } from '../Spinner';

describe('Spinner', () => {
  test('render', () => {
    const wrapper = render(<Spinner />);
    expect(wrapper.container).toMatchSnapshot();
  });
});
