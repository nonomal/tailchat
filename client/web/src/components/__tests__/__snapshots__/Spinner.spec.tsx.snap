// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner render 1`] = `
<div>
  [iconify icon="mdi:loading"]
</div>
`;
